packages/augur-core/source/libraries/NetworkConfiguration.ts

```diff
@@ -18,7 +18,7 @@ export const NETWORKS = [
   "kovan",
   "thunder",
   "testrpc"
-] as const;
```

Comment:
This is typescript 3.4 syntax. Shouldn't *need* to be removed but would fail if the wrong version of typescript was used to build the project.

Reply:
It was causing an error in my builds but inconsistently. Was very strange and this fixed it so...